code/modules/surgery/organs/lungs.dm

```diff
@@ -252,9 +252,44 @@
 				H.adjustBrainLoss(3)
 		else if(bz_pp > 0.01)
 			H.hallucination += 5//Removed at 2 per tick so this will slowly build up
+
+
+	// Tritium
+		var/trit_pp = breath.get_breath_partial_pressure(breath_gases["tritium"][MOLES])
+		if (trit_pp > 50)
+			H.radiation += trit_pp/2 //If you're breathing in half an atmosphere of radioactive gas, you fucked up.
+		else
+			H.radiation += trit_pp/10
+
+	//Brown Gas
+		var/brown_pp = breath.get_breath_partial_pressure(breath_gases["browns"][MOLES])
+		if (prob(brown_pp))
+			to_chat(H, "<span class='alert'>Your mouth feels like it's burning!</span>")
+		if (brown_pp >40)
+			H.emote("gasp")
+			H.adjustFireLoss(10)
+			if (prob(brown_pp/2))
+				to_chat(H, "<span class='alert'>Your throat closes up!</span>")
+				H.silent = max(H.silent, 3)
+		else
+			H.adjustFireLoss(brown_pp/4)
+		gas_breathed = breath_gases["browns"][MOLES]
+		if (gas_breathed > GAS_STIM_MINIMUM)
+			H.status_flags |= GOTTAGOFAST
+		else
+			H.status_flags &= ~GOTTAGOFAST
+
+		breath_gases["browns"][MOLES]-=gas_breathed
+		gas_breathed = 0
+	//Stimulum
+		gas_breathed = breath_gases["stim"][MOLES]
+		if (gas_breathed > GAS_STIM_MINIMUM)
+			H.status_flags |= GOTTAGOFAST
+		else
+			H.status_flags &= ~GOTTAGOFAST
```

Comment:
You'll remove the flag added from brown gas or other sources this way, same above.

Reply:
Good point. Do you know a way I could avoid that?

Reply:
Yeah this'll get fucked by chem code too